stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx

```diff
@@ -57,7 +57,7 @@ export const ContentCardWithAndWithoutImage = ({
                     : (
                       <></>
                     )}
-                  <Ctalink label={item.button} Type={item.btnType} />
+                  <span className="cta__link cta--space">{item.button}</span>
```

Comment:
Hey @neelesh-tippe  can we replace this mark with cta component eg :
`<Ctalink label={item.button} Type="Space" Emphasize="false"/>`

Reply:
Done, can you please review and merge this PR